static/bidder-info/bliink.yaml

```diff
@@ -0,0 +1,21 @@
+maintainer:
+  email: "support@bliink.io"
+gvlVendorID: 658
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - audio
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - audio
+      - native
+userSync:
+  redirect:
+    url: "https://cookiesync.api.bliink.io/getuid?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&url={{.RedirectURL}}"
+    userMacro: "$UID"
```

Comment:
Please make sure your userSync URL works.
I tested it this way: 
Default PBS URL : https://prebid.adnxs.com/pbs/v1/setuid?bidder=bliink&gdpr=&gdpr_consent=&f=i&uid=$UID -  with bidder=bliink and uid=$UID (user macro) 
Encoded: 
https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID
Final UserSync URL: https://cookiesync.api.bliink.io/getuid?gdpr=&consent=&us_privacy=&url=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID - returns http 400. Ideally should return 302 redirect to PBS with UID replaced with real ID.

Reply:
Hi, there is a 400 error because no cookie is set in your browser. 
We have several options for not returning this error code: 
 • We can redirect to your url with a blank cookie
 • We can set a cookie to your browser and redirect with this new created cookie

Thank you

Reply:
This should not depend on cookies set to browser. @SyntaxNode please advice.

Reply:
Discussed with the team. Please disregard my previous comment. 
Could you please return http 200 instead of http 400 so Prebid Server can handle it properly.

Reply:
Hi, we've fixed UserSync code returns, this should work now.

Reply:
Thank you, it now works as expected!